roiextractors/extractors/schnitzerextractor/extractsegmentationextractor.py

```diff
@@ -2,7 +2,9 @@
 import h5py
 from ...segmentationextractor import SegmentationExtractor
 from lazy_ops import DatasetView
-from ...extraction_tools import _pixel_mask_extractor
+from roiextractors.extraction_tools import _pixel_mask_extractor
```

Comment:
@Saksham20 I think we settled on relative imports!

Reply:
I had made relative imports in another branch which was merged #33 .
Since these PRs were made at the same time, there will be some which will not have the implementation/corrections of the other.